JNLP examples first version (#28)
leaningtech/labs.leaningtech.com: src/content/docs/cheerpj3/13-examples/02-jnlp.mdx

- **The JAR files:** Given by the `<jar>` tag in the `<resources>` section.
- **The class name:** Given by `main-class` under the `<applet-desc>` tag. This tag also indicates that the app is an applet.

```xml title= "PitchApplet.jnlp" {5, 21, 23}
<?xml version="1.0" encoding="utf-8"?>
<!-- JNLP File for Pitch applet -->
<jnlp
  spec="1.0+"
  codebase="https://raw.githubusercontent.com/leaningtech/cheerpj-example-applet/main/"

  href="PitchApplet.jnlp">
  <information>
    <title>Pitch</title>
    <vendor>NASA Glenn Research Center</vendor>
    <homepage href="https://www.grc.nasa.gov/WWW/K-12/airplane/"/>
    <description>Pitch motion simulator</description>
    <description kind="short">Beginner's Guide to Aeronautics - Pitch motion simulator written in Java</description>
    <offline-allowed/>
  </information>
  <security>
      <all-permissions/>
  </security>
  <resources>
    <j2se version="1.4+" initial-heap-size="30m" max-heap-size="300m" />
    <jar href="Pitch.jar"/>
  </resources>
  <applet-desc main-class="Pitchview" width="300" height="500"/>
</jnlp>
```

### 3. Download the applet file

> You will download the applet JAR files by manually building the full URL and pasting it in the browser navigation bar. This is done by appending the `codebase` URL and the `jar` URL.

Reviewer: ```suggestion
Download the applet JAR files by manually building the full URL and pasting it in the browser navigation bar. This is done by concatenating the `codebase` URL and the `jar` URL.
```

For example:

```
https://raw.githubusercontent.com/leaningtech/cheerpj-example-applet/main/Pitch.jar

```

### 3. Create a project directory

> Create a directory where all the files will live. You can choose any name such `cheerpj-example-applet`:

Reviewer: ```suggestion
Create a directory where all the files will live. You can choose any name, such as `cheerpj-example-applet`:
```
